{"name": "test_exception_match", "status": "broken", "statusDetails": {"message": "TypeError: 传入的参数不是整数", "trace": "self = <test_leap_year.TestAssert object at 0x0000028E19A64288>\n\n    def test_exception_match(self):\n        with pytest.raises(ValueError, match='公元33元年是从公元一年开始') as excinfo:\n>           is_leap_year('y')\n\ntest_leap_year.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nyear = 'y'\n\n    def is_leap_year(year):\n        if isinstance(year, int) is not True:\n>           raise TypeError(\"传入的参数不是整数\")\nE           TypeError: 传入的参数不是整数\n\ntest_leap_year.py:15: TypeError"}, "start": 1588053929014, "stop": 1588053929014, "uuid": "75118e1e-d1be-4668-bc25-265350952285", "historyId": "0bc83563d379e090cbbc3dd4fb9c58b2", "testCaseId": "7ad40325dbda3229eb4244c14bc09640", "fullName": "test_leap_year.TestAssert#test_exception_match", "labels": [{"name": "suite", "value": "test_leap_year"}, {"name": "subSuite", "value": "TestAssert"}, {"name": "host", "value": "PC-BFD8TY2"}, {"name": "thread", "value": "404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_leap_year"}]}